src/assets/assetsUtil.ts

```diff
@@ -93,5 +93,5 @@ export const formatIconUrlWithProxy = ({
   chainId: string;
   tokenAddress: string;
 }) => {
-  return `https://static.metaswap.codefi.network/api/v1/tokenIcons/${chainId}/${tokenAddress}.png`;
+  return `https://static.metaswap.codefi.network/api/v1/tokenIcons/${chainId}/${tokenAddress.toLowerCase()}.png`;
```

Comment:
The description states that the token address is expected to already be lower case.

Do you know what format the token address is in when this is called? e.g. is it always mixed-case, or is it sometimes lower-case and sometimes mixed-case?